components/servo/lib.rs

```diff
@@ -1045,28 +998,71 @@ fn create_sandbox() {
     panic!("Sandboxing is not supported on Windows, iOS, ARM targets and android.");
 }
 
-/// Implements GLContextDispatcher to dispatch functions from GLContext threads to the main thread's event loop.
-/// It's used in Windows to allow WGL GLContext sharing.
-pub struct MainThreadDispatcher {
-    compositor_proxy: CompositorProxy,
-}
-
-impl MainThreadDispatcher {
-    fn new(proxy: CompositorProxy) -> Self {
-        Self {
-            compositor_proxy: proxy,
+// Initializes the WebGL thread.
+fn create_webgl_threads<W>(
+    window: &W,
+    webrender: &mut webrender::Renderer,
+    webrender_api_sender: webrender_api::RenderApiSender,
+    webvr_compositor: Option<Box<WebVRCompositorHandler>>,
+    webxr_main_thread: &mut webxr_api::MainThreadRegistry,
+    external_image_handlers: &mut WebrenderExternalImageHandlers,
+    external_images: Arc<Mutex<WebrenderExternalImageRegistry>>,
+) -> WebGLThreads
+where
+    W: WindowMethods + 'static + ?Sized,
+{
+    // Create a `surfman` device and context.
+    //
+    // FIXME(pcwalton): The `NoopContextReleaser` is an unsafe hack to get around the fact that
```

Comment:
This comment refers to code that was changed, I believe.

Reply:
Removed